code/_helpers/medical_scans.dm

```diff
@@ -70,6 +69,7 @@
 		O["brute_dam"] = E.brute_dam
 		O["burn_dam"] = E.burn_dam
 		O["scan_results"] = E.get_scan_results(tag)
+		O["tumors"] = E.has_growths()
```

Comment:
Looking forward to seeing Chinsky add Space Cancer

Reply:
It's been on my todo for ages!